### ISSUE TITLE
HDMI Audio not working reliably.

### ISSUE THREAD

**Issue by anand7825** [shuai620/NUC7i7BNH#15]
I have an annoying problem with my NUC7i5BNK. I used the EFI with OpenCore 0.7.7 with Monterrey. I have my NUC connected to a Samsung TV using HDMI. It used to work fantastic with BigSur with 0.6.5 version of OpenCore EFI.
The problem I am running into is HDMI Audio. The first time I start the NUC with the TV ON, there is video but no sound device(Displayport).When I shut down the TV and Turn it ON, the sound device automatically appears and I am able to listen to sound. 
I read the Dortania guide and looked up my device id 0x5926 from https://ark.intel.com/content/www/us/en/ark/products/97539/intel-core-i57260u-processor-4m-cache-up-to-3-40-ghz.html
I modified the config.plist and added the new device id. This did not fix the problem. 

Any ideas??

**Comment by shuai620** [comment]
Yes, not only HDMI audio but also tpye-C audio needs reconnection to work properly.
I don't have a clue yet.

**Comment by shuai620** [comment]
Problem solved. See new update of "FakePCIID.kext".
